client.go

```diff
@@ -1872,3 +2178,29 @@ func (cl *Client) banPeerIP(ip net.IP) {
 	}
 	cl.badPeerIPs[ip.String()] = struct{}{}
 }
+
+func (cl *Client) Stats() (downloaded int64, uploaded int64) {
+	cl.mu.Lock()
+	defer cl.mu.Unlock()
+	downloaded = cl.downloaded
+	uploaded = cl.uploaded
+	return
+}
```

Comment:
This would be better as a block. Return a ClientStats struct. There will be more values.